### ISSUE TITLE
Popup showing behind the page

### ISSUE THREAD

**Issue by vnihoul77** [joeattardi/picmo#230]
Hi,
I am using the native code from the popup picker, and the popup seems to appear behind the page. [Demo](https://plugbee-iframes.bubbleapps.io/version-test/emoji-picker2)

This is my code in my HTML tag:
```
<script type="module">
import { createPopup } from 'https://unpkg.com/@picmo/popup-picker@latest/dist/index.js?module';
window.createPopup = createPopup
</script>
```
And in my JS element:
```
window.addEventListener('load', (event) => {
    const triggerButton = document.getElementById('button');

    // Create the picker
    const picker = createPopup({

      className: "pluginpicker",
      hideOnClickOutside: true,
      hideOnEmojiSelect: true,
      hideOnEscape: true,
      showCloseButton: true,
      position: 'bottom-start', //https://floating-ui.com/docs/computeposition#placement
      //position: 'auto';

    }, {
      // The element that triggers the popup
      triggerElement: triggerButton,

      // The element to position the picker relative to - often this is also the trigger element,
      referenceElement: triggerButton,

    });

    // The picker emits an event when an emoji is selected. Do with it as you will!
    picker.addEventListener('emoji:select', event => {
      console.log('Emoji selected:', event.emoji);
    });

    triggerButton.addEventListener('click', event => {
      picker.toggle()
    });
}
});
```

I tried adding a custom class with a z-index but then the close button appears behind the popup. [See here](https://plugbee-iframes.bubbleapps.io/version-test/emoji-picker)

```
<style>
pluginpicker {
  z-index: 1001; // Show over mat-dialog
}
.pluginpicker {
  z-index: 1001; // Show over mat-dialog
}
</style>
```

Any hint? 🤔 
Thank!

**Comment by joeattardi** [comment]
I tried to go to your example link but it threw a big error in the console. If you can set up a CodePen, CodeSandbox, or similar, I am happy to take a further look. Z-index is tricky, it's not always as simple as just setting a higher `z-index` value because there may be a higher stacking context.

**Comment by vnihoul77** [comment]
Sorry, I was updating the code when you visited, trying to solve the issue.
Unfortunately, I am not able to reproduce the bug in CodePen.
The environment I am working on is a bit specific because I am trying to integrate your library as a Bubble plugin as the editor limits me.

Is there a way I could manually setup only the close button to be `z-index = 1002`? 
This way I could set up the popup to be 1001 and the close button to be higher.

**Comment by joeattardi** [comment]
I just published a new release (5.5.1) that should make this easier for you to override styles.

The close button now has a simple CSS selector (`.popupContainer .closeButton`) that you should be able to select and override the z-index for the button. Hope this helps!

**Comment by vnihoul77** [comment]
It does, thanks!
